spec/factories/review.rb

```diff
@@ -26,8 +26,9 @@
     end
 
     factory :twenty_four_month_review do
-      review_type "24-Month"
-      review_date { 24.months.from_now }
+      review_type '24-Month'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.